vis_services/lib/paper_network.py

```diff
@@ -201,22 +201,22 @@ def get_papernetwork(solr_data, max_groups, weighted=True, equalization=False, d
               of the number of references in the linked nodes.
     '''
     # Get get paper list from the Solr data
-    papers_list = map(lambda a: a['bibcode'], solr_data)
+    papers_list = [a['bibcode'] for a in solr_data]
     number_of_papers = len(papers_list)
     # First construct the reference dictionary, and a unique list of cited papers
     reference_dictionary = _get_reference_mapping(solr_data)
     # From now on we'll only work with publications that actually have references
-    papers = reference_dictionary.keys()
+    papers = list(reference_dictionary.keys())
```

Comment:
No need for `list` here.

Reply:
keys function returns a dict_keys object which isn't a list and that causes problems